docs/building-spark.md

```diff
@@ -259,6 +259,14 @@ or
 Java 8 tests are automatically enabled when a Java 8 JDK is detected.
 If you have JDK 8 installed but it is not the system default, you can set JAVA_HOME to point to JDK 8 before running the tests.
 
+## PySpark pip installable
+
+If you are building Spark for use in a Python environment and you wish to pip install it, you will first need to build the Spark JARs as described above. Then you can construct an sdist package suitable for setup.py and pip installable package.
+
+    cd python; python setup.py sdist
```

Comment:
Just to confirm, if I run this:

```
./dev/make-distribution.sh --pip
```

It should take care of both building the right JARs _and_ building the Python package.

Then I just run:

```
pip install -e ./python/
```

to install Spark into my Python environment.

Is that all correct?